apps/open-justice/aleph.env

```diff
@@ -0,0 +1,103 @@
+# Aleph environment configuration
+#
+# This file is loaded by docker-compose and transformed into a set of
+# environment variables inside the containers. These are, in turn, parsed
+# by aleph and used to configure the system.
+
+# Random string:
+ALEPH_SECRET_KEY={{aleph_secret_key}}
+
+# Visible instance name in the UI
+ALEPH_APP_TITLE=Open Justice
+# Name needs to be a slug, as it is used e.g. for the ES index, SQS queue name:
+ALEPH_APP_NAME=open-justice
+ALEPH_UI_URL=http://openjustice.openup.org.za:8080/
```

Comment:
hmmm we need SSL... I wonder what I did for that before - I think we shouldn't have anything sensitive in there or have people create accounts that matter until we set SSL up